src/utils/display-text/format.ts

```diff
@@ -109,5 +109,13 @@ export function calculateProfit(
   return isLong ? profitLong : profitShort;
 }
 
-export const stringToFixedPrecision = (value: string, precision: number) =>
-  parseFloat(value).toFixed(precision);
+export const countDecimals = value => {
+  if (Math.floor(value) === value || !value.split('.')[1]) return 0;
```

Comment:
I would extract `value.split('.')[1]` to a variable (e.g. `decimalPart`) and use it here as `!decimalPart`.